bigbluebutton-html5/imports/ui/components/user-list/service.js

```diff
@@ -311,6 +311,8 @@ const removeUser = (userId) => {
 
 const toggleVoice = (userId) => { makeCall('toggleVoice', userId); };
```

Comment:
Isn't easier to just do `const toggleVoice = (userId) => { userId === Auth.userID ? makeCall('toggleSelfVoice') : makeCall('toggleVoice', userId); };`

Instead of creating a `toggleSelfVoice` and passing it down? :thinking:

Reply:
Yeah, I agree. It's easier to put this logic into `toggleVoice`. I pushed the modifications.